### ISSUE TITLE
error loading file - "Not a diagram file" when I tried to open it, file corrupted after sudden shut-down

### ISSUE THREAD

**Issue by djt-djt** [jgraph/drawio#4462]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

If you are technical, you should reporting bugs along the lines of https://marker.io/blog/how-to-write-bug-report. If you are not technical, we will make allowances, please try to make an effort to understand the process.

**Describe the bug**
Error upon loading drawio file - "Not a diagram file" when I tried to open it, file corrupted after sudden shut-down

**To Reproduce**
1. Go to 'Diagrams.net'
2. Click on 'open existing diagram' from local device
4. See error "Not a diagram file" 

**Expected behavior**
Should open existing diagram

**Screenshots**
![image](https://github.com/jgraph/drawio/assets/172054349/652631af-6086-4bf2-af8d-152d49351880)

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 24.5.2

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser [firefox]
 - Browser Version [126.0.1]

**Smartphone (please complete the following information):**

**I tested the problem in incognito/private mode with all browser extensions switched off, write "yes" below:**
-

**Additional context**
Diagram file corrupted upon sudden shut down, opening the source , there are only 'NULL's inside, still it is not empty, file size 3.4 MB.

**Comment by davidjgraph** [comment]
Is there a .bkp file in the same folder?

**Comment by djt-djt** [comment]
Not at all, checked ..Unfortunately windows file history is disabled by group policy.  Sent from Mail for Windows From: David BensonSent: Friday, June 7, 2024 17:47To: jgraph/drawioCc: djt-djt; AuthorSubject: Re: [jgraph/drawio] error loading file - "Not a diagram file" when I tried to open it, file corrupted after sudden shut-down (Issue #4462) Is there a .bkp file in the same folder?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by djt-djt** [comment]
ok, thanks ..
if there is no chance to repair , will close the case.

**Comment by davidjgraph** [comment]
Unfortunately, not. Sorry for the inconvenience.
